krun/results.py

```diff
@@ -0,0 +1,100 @@
+from krun.audit import Audit
+from krun.config import Config
+
+import bz2  # decent enough compression with Python 2.7 compatibility.
+import json
+
+
+class Results(object):
+    """Results of a Krun benchmarking session.
+    Can be serialised to disk.
+    """
+    def __init__(self, config, platform, results_file=None):
```

Comment:
I suppose you could add one there too, but I was referring to the body of the constructor. Just to separate out the concepts.